docs/node-operators/migration-guides/update-network-with-checkpoint.md

```diff
@@ -0,0 +1,265 @@
+4. Approved PR must be merged by one of the validators.
+
+### 6. Download new genesis file
+After creating a backup and preparing a new genesis file, put it on your server. All the validators **must** use the same genesis file. 
+
+1. Download genesis
+2. Remove old genesis at `<TENDERMINT-HOME>/config/genesis.json`
+3. Save new, downloaded genesis to `<TENDERMINT-HOME>/config/genesis.json`
+4. Verify genesis - see example below
+
+#### Example workflow
+An example workflow for reviewing the genesis file may look like following:
+
+```bash 
+# Download genesis
+wget https://raw.githubusercontent.com/vegaprotocol/networks/master/mainnet1/genesis.json
+
+# Move old genesis to a different location
+cp <TENDERMINT-HOME>/config/genesis.json <TENDERMINT-HOME>/config/genesis.json.bk
+
+# Copy genesis to its final location
+cp ./genesis.json <TENDERMINT-HOME>/config/genesis.json
+
+# Verify genesis
+<VEGA-BIN> verify genesis <TENDERMINT-HOME>/config/genesis.json
+```
+
+### 7. Update Vega core config
+
+:::note Manual process
+This step may be time consuming, as there is no automation and it needs to be done by hand. 
+:::
+
+There are a few ways to update your existing Vega config. The most practical way is to see what changed in the Vega config between versions, as follows:
+
+1. Generate the Vega node in a temporary location: `<VEGA-BIN> init --home /tmp/vega-home <TYPE>`. When the terminal asks you about passphrases, type anything. You are interested only in the `config.toml` file. The `<TYPE>` may be different depending on the configuration you are running:
+    - a. `validator` - if you are running only Vega core without data node, e.g.: `<VEGA-BIN> init --home /tmp/vega-home validator`
+    - b. `full` - if you are running Vega core with a data node, e.g.: `<VEGA-BIN> init --home /tmp/vega-home full`
+2. Compare the old config with the generated one: `diff <VEGA-NETWORK-HOME>/config/node/config.toml /tmp/vega-home/config/node/config.toml`
+3. Update your `<VEGA-NETWORK-HOME>/config/node/config.toml` file based on the above diff
+
+:::warning Config parameters
+We strongly recommend you read the list of configuration changes in the [upgrading file ↗](https://github.com/vegaprotocol/vega/blob/develop/UPGRADING.md#configuration-changes) to understand what config parameters and sections have changed.
+
+You are responsible for deciding what parameters you want to use. `vega init` generates a config with default values. Values in your config may be changed intentionally. Review and prepare your config carefully.
+:::
+
+### 8. Update Tendermint config
+
+:::note Manual process
+This step may be time consuming, as there is no automation and it needs to be done by hand. 
+:::
+
+The procedure is very similar to updating the Vega config. We recommend you read the [documentation for running Tendermint in production ↗](https://docs.tendermint.com/v0.33/tendermint-core/running-in-production.html)

```

Comment:
Maybe also add a link to this: https://github.com/vegaprotocol/vega/blob/develop/UPGRADING.md#tendermint

Reply:
Thank you added